docs/developer_guide.md

```diff
@@ -36,15 +36,101 @@ go mod vendor
 ```
 
 ## Running the Operator Locally
+
 Running the operator locally (as opposed to deploying it on a K8s cluster) is convenient for debugging/development.
 
 ### 1. Preliminary
 
+#### Minikube Install
+
 Install [minikube](https://kubernetes.io/docs/tasks/tools/) on your loptop.
-And you can start minikube by the command
+
+#### Minikube with GPU Support
+
+To enable GPU support follow the doc to install [containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) and [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-docker) then enable [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin#preparing-your-gpu-nodes) and test your GPU by the official [gpu-pod](https://github.com/NVIDIA/k8s-device-plugin#running-gpu-jobs)
+
+It is highly recommended to have more than one GPU resource in your workspace. However, there is still a workaround to divide your singie GPU resource into multiple ones. For this, enable [shared-access-to-gpus with CUDA Time-Slicing](https://github.com/NVIDIA/k8s-device-plugin#shared-access-to-gpus-with-cuda-time-slicing) to get more GPU resources.
```

Comment:
`singie GPU` -> `single GPU`.

The line is too long and split it to lines whose length < 80.

Reply:
Okay, I will update this in a new commit.